### ISSUE TITLE
Add annotations for reference genome build/assembly

### ISSUE THREAD

**Issue by jaeddy** [Sage-Bionetworks/synapseAnnotations#120]
For any processed data, it'd be great to have some tags indicating genome build (e.g., hg19 vs. GRCh38), possibly as fine-grained as version (e.g., patch release 10 vs. patch release 9). I suggest adding a key to minified based on the GA4GH class [`ReferenceSet`](http://ga4gh-schemas.readthedocs.io/en/latest/schemas/references.proto.html#protobuf.ReferenceSet).

For example:

```
[
  ...
  {
    "name": "referenceSet",
    "description": "",
    "columnType": "STRING",
    "maximumSize": 250,
    "enumValues": [
      {
        "value": "GRCh38", 
        "description": "Genome Reference Consortium Human Build 38", 
        "source": "https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.26"
      },
  ...
]
```

Accounting for version could get complicated, as you might have to deal with patch number as well as contributing source (e.g., GRC vs. UCSC). Gets even trickier if you include reference _transcriptomes_ (i.e., FASTA sequences derived from gene models in Ensembl, RefSeq, GENCODE, etc. that all have their own release versions as well). Would need to give this some more thought...

**Comment by kdaily** [comment]
@jaeddy if this is still required please open a PR with changes to `ngs.json`, thanks!

**Comment by sgosline** [comment]
Let's discuss this in the context of scRNA-Seq annotations

**Comment by allaway** [comment]
This is related to the issue we discussed today, #350

**Comment by sgosline** [comment]
It will go into curatedData.json for now.

**Comment by kelshmo** [comment]
Do we have consensus of how to annotate with this key? I believe it makes sense to add values for both Ensembl and UCSC ref genomes.

**Comment by karawoo** [comment]
I believe this was closed in #439
